### ISSUE TITLE
unusual error when presenting viewcontroller

### ISSUE THREAD

**Issue by wdcurry** [chiunam/CTAssetsPickerController#176]
99.9% of my installs are working with CTAssetsPickerController. But i have one beta tester who is seeing his app barf with [self presentViewController:picker animated:YES completion:nil]; ..

-[UILabel length]: unrecognized selector sent to instance 0x12fd47950
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[UILabel length]: unrecognized selector sent to instance 0x12fd47950'

I don't have access to his device long enough to break down into the pod code to see this happen real-time. Just wondering if anyone has a clue from this minimal info. I tested the exact same device under my control and it works fine.

thx..

**Comment by chiunam** [comment]
@wdcurry Thanks for reporting.  Did a quick search and cannot find any code of reading `length` from  label. Not sure why it happens. Let's keep in view.

**Comment by zavsby** [comment]
I faced with that crash too (on iOS 8.0.2, iPhone 6 Plus).
It looks like that a problem is inside CTAssetsPickerNoAssetsView class, updateConstraints method. 
The crash was caused by these two lines
`[self autoPinEdgeToSuperviewMargin:ALEdgeLeading];`
`[self autoPinEdgeToSuperviewMargin:ALEdgeTrailing];`
By replacing it with layout without using margins (8px leading and trailing insets) it works without crash.
Hope it helps.
